private_sharing/api_views.py

```diff
@@ -235,7 +285,10 @@ class ProjectFileDirectUploadView(ProjectFormBaseView):
     form_class = DirectUploadDataFileForm
 
     def post(self, request):
-        super(ProjectFileDirectUploadView, self).post(request)
+        super().post(request)
+
+        if not self.good_datatypes:
+            return HttpResponseForbidden()  # Did not include a properly formed datatype
```

Comment:
This should be resulting in a serializer error, as with other API views, e.g.:
https://github.com/OpenHumans/open-humans/blob/master/private_sharing/api_views.py#L160